.circleci/config.yml

```diff
@@ -118,8 +127,18 @@ workflows:
             tags:
               only: /^v+(\d)*$/
       - push-assets-to-github:
-          name: "Upload Node 14 plugin"
-          node-version: '14'
+          name: "Upload Node 18 plugin"
+          node-version: '18'
```

Comment:
It looks like there are two sets of the v18 `push-assets-to-github` task, and one should be removed?